internal/kubernetes/access.go

```diff
@@ -23,7 +24,12 @@ func NewAccessValidator(config aws.Config) AccessValidator {
 func (a AccessValidator) Run(ctx context.Context, informer validation.Informer, node *api.NodeConfig) error {
 	// APIServerEndpoint, CertificateAuthority are required for the validations we want to run here but are optional in the config
 	// When not specified, we need to read them from the EKS API.
-	cluster, err := eks.ReadClusterDetails(ctx, a.aws, node)
+	var cluster *api.ClusterDetails
+	err := retry.NetworkRequest(ctx, func(ctx context.Context) error {
+		var err error
+		cluster, err = eks.ReadClusterDetails(ctx, a.aws, node)
```

Comment:
I'm not sure we need this here. The aws client already handles retries
@jaxesn

Reply:
Yeah we can remove this. We use the defaults on the aws config, which are up to 3 retries, which I think is fine for nodeadm commands (in the e2e tests we bump it to like 20 since we are making so many concurrent requests).  If we do see issues tho we can tweak the retry config on the aws config.